File: Safari-609.1.11.1/LayoutTests/http/tests/navigation/resources/goback-with-policydelegate.php
<html>
<head>
<script>

function loaded()
{
    if (window.testRunner)
        testRunner.setCustomPolicyDelegate(true, true);

    window.history.back();
}

</script>
</head>
<body onload="loaded();">
This page turns on DRT's custom policy delegate then navigates back a page.
If you're running the test in the browser, you should not get the form resubmission nag.
</body>
</html>

File: LayoutTests/http/tests/navigation/resources/goback-with-policydelegate.py
#!/usr/bin/env python3

import sys

sys.stdout.write(
    'Content-Type: text/html\r\n\r\n'
    '<html>\n'
    '<head>\n'
    '<script>\n'
    '\n'
    'function loaded()\n'
    '{\n'
    '    if (window.testRunner)\n'
    '        testRunner.setCustomPolicyDelegate(true, true);\n'
    '\n'
    '    window.history.back();\n'
    '}\n'
    '\n'
    '</script>\n'
    '</head>\n'
    '<body onload="loaded();">\n'
    'This page turns on DRT\'s custom policy delegate then navigates back a page.\n'
    'If you\'re running the test in the browser, you should not get the form resubmission nag.\n'
    '</body>\n'
    '</html>\n'
)